android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -87,13 +99,21 @@
     <!-- Title of the time picker dialog. -->
     <string name="mozac_feature_prompts_set_time">Set time</string>
     <!-- Option in expanded select login prompt that links to login settings -->
-    <string name="mozac_feature_prompts_manage_logins">Manage logins</string>
+    <string name="mozac_feature_prompts_manage_logins" moz:removedIn="123" tools:ignore="UnusedResources">Manage logins</string>
```

Comment:
```suggestion
    <string name="mozac_feature_prompts_manage_logins" moz:removedIn="124" tools:ignore="UnusedResources">Manage logins</string>
```
@sarah541 Since the cut has already happened, we need to bump all `moz:removedIn` to 124.